Web/Edubase.Services/Nomenclature/NomenclatureService.cs

```diff
@@ -2,6 +2,7 @@
 using Edubase.Common.Text;
 using Edubase.Services.Enums;
 using System;
+using System.Diagnostics;
```

Comment:
Introduction of a new, unused, `using` - a legacy from intermediate refactoring and is to be removed, perhaps?

Reply:
will have a look at this

Reply:
removed the unused using - it was from intermediate steps